### PR TITLE
Added arquivo.pt archive support

### PR DESCRIPTION
**Pull request by dhruv282**
- Added arquivo.pt radio button
- Updated ReadMe file appropriately.
- Fixed URI-M and URI-T identification on the front-end

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ Supported server arguments are as follows:
 To query the server instance generated using your browser visit `http://localhost:3000/alsummarizedtimemap/archiveIt/1068/4/[histogram | stats | summary]/[from]/[to]/http://4genderjustice.org/`, which has the attributes path as `primesource/ci/role/from/to/URI-R` substitute the URI-R to request a different site's summarization.
 
 ### Parameter Definitions:
-- `primesource` gets the value of 'archiveIt' or 'internetarchive' as to let the service know which is the primary source.
+- `primesource` gets the value of 'archiveIt', 'arquivopt', or 'internetarchive' as to let the service know which is the primary source.
```
Comment (ibnesayeed) [inline README.md:71]:
```suggestion
- `primesource` gets the value of 'archiveit', 'arquivopt', or 'internetarchive' as to let the service know which is the primary source.
```

**Review by ibnesayeed — CHANGES_REQUESTED** [review]

**Review by ibnesayeed — COMMENTED** [review]
LGTM! Let me know when you are ready for it to be merged.

**Review by ibnesayeed — APPROVED** [review]
Forgot to mark it approved.

**Comment by dhruv282** [comment]
It's ready to be merged, thank you!!

**Comment by weiglemc** [comment]
Thanks all!

**Pull request merged**